src/popup/components/Modals/TransferReceive.vue

```diff
@@ -31,7 +31,10 @@
           :copied="copied"
           @click="copyAddress()"
         >
-          <Scrollable class="address-scrollable-area">
+          <Scrollable
+            class="address-scrollable-area"
+            hide-scrollbar
```

Comment:
On figma designs this place should have the scrollbar visible.

Reply:
https://www.figma.com/file/3oGLWzSH0oJljo4RETZtur/Superhero-Wallet-UI-(%E2%9C%94%EF%B8%8FUpdated)?node-id=11533%3A143300

Reply:
I asked Tsvetan about this and he confirmed what I stated.